content/id/docs/tasks/job/automated-tasks-with-cron-jobs.md

```diff
@@ -0,0 +1,214 @@
+Mengamati Job yang akan dibuat sekitar satu menit:
+
+```shell
+kubectl get Job --watch
+```
+Keluaran akan mirip dengan ini:
+
+```
+NAME               COMPLETIONS   DURATION   AGE
+hello-4111706356   0/1                      0s
+hello-4111706356   0/1           0s         0s
+hello-4111706356   1/1           5s         5s
+```
+
+Sekarang kamu telah melihat satu Job berjalan yang dijadwalkan oleh "hello" CronJob.
+Kamu dapat berhenti mengamati Job dan melihat CronJob lagi untuk melihat CronJob menjadwalkan sebuah Job:
+
+```shell
+kubectl get cronJob hello
+```
+Keluaran akan mirip dengan ini:
+
+```
+NAME    SCHEDULE      SUSPEND   ACTIVE   LAST SCHEDULE   AGE
+hello   */1 * * * *   False     0        50s             75s
+```
+
+Kamu dapat melihat bahwa CronJob `hello` berhasil menjadwalkan sebuah Job pada waktu yang ditentukan dalam `LAST SCHEDULE`. Saat ini ada 0 Job yang aktif,  berarti sebuah Job telah selesai atau gagal.
+
+Sekarang, temukan Pod yang dibuat oleh jadwal Job terakhir dan melihat keluaran bawaan dari salah satu Pod
+
+{{< note >}}
+Nama Job dan nama Pod itu berbeda.
+{{< /note >}}
+
+```shell
+# Ganti "hello-4111706356" dengan nama Job di sistem kamu
+pods=$(kubectl get pods --selector=Job-name=hello-4111706356 --output=jsonpath={.items[*].metadata.name})
+```
+Menampilkan log sebuah Pod:
+
+```shell
+kubectl logs $pods
+```
+Keluaran akan mirip dengan ini:
+
+```
+Fri Feb 22 11:02:09 UTC 2019
+Hello dari klaster Kubernetes
+```
+
+## Menghapus sebuah CronJob
+
+Ketika kamu tidak membutuhkan sebuah CronJob lagi, kamu dapat megnhapusnya dengan perintah `kubectl delete cronJob <nama cronJob>`:
+
+```shell
+kubectl delete cronJob hello
+```
+
+Menghapus CronJob akan menghapus semua Job dan Pod yang telah terbuat dan menghentikanya dari pembuatan Job tambahan.
+Kamu dapat membaca lebih lanjut tentang menghapus Job di [_garbage collection_](/docs/concepts/workloads/controllers/garbage-collection/).
+
+## Menulis Speifikasi Sebuah Cron
+
+Seperti semua konfigurasi Kubernetes, sebuah CronJob membutuhkan bidang `apiVersion`, `kind`, dan `metadata`. Untuk informasi
+umum tentang bekerja dengan berkas konfigurasi, lihat dokumentasi [men-deploy aplikasi](/docs/user-guide/deploying-applications),
+dan [mengunakan kubectl untuk manajemen sumber daya](/docs/user-guide/working-with-resources).
+
+Sebuah konfigurasi CronJob juga membutuhkan sebuah [bagian `.spec`](https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status).
+
+{{< note >}}
+Semua modifikasi pada sebuah CronJob, terutama `.spec`, akan diterapkan pada proses berikut.
+{{< /note >}}
+
+### Penjadwalan
+
+`.spec.schedule` adalah bidang yang wajib diisi dari sebuah `.spec`
+Dibutuhkan sebuah fromat string [Cron](https://en.wikipedia.org/wiki/Cron), misalnya `0 * * * *` atau `@hourly`, sebagai jadwal Job untuk dibuat dan dieksekusi.
```

Comment:
Maybe provide a translation for “hourly”?

Reply:
I think hourly is crontab syntax right? so no need to translate @sftim

Reply:
No need to _translate_ `@hourly`, but does it make sense to _explain_? (I might have missed something!)

Reply:
which part is missing?, I think that @hourly have context on example of cron syntax or should I describe more what is @hourly means?